extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -68,7 +75,7 @@ class AmpFitText extends AMP.BaseElement {
       left: 0,
       zIndex: 1,
       visibility: 'hidden',
-      lineHeight: `${LINE_HEIGHT_EM_}em`,
+      // lineHeight: `${LINE_HEIGHT_EM_}em`,
```

Comment:
> Instead of using the constant here, we should copy it from the actual element
i like this. would you suggest we create an attribute on the AMP element? or extract it from `getComputedStyle`?